parley/src/font.rs

```diff
@@ -6,9 +6,18 @@ use fontique::Collection;
 #[cfg(feature = "std")]
 use fontique::SourceCache;
 
+/// A font database/cache (wrapper around a the [fontique] crate)
```

Comment:
a Fontique [`Collection`]?

Reply:
"a the" ... choose one!

Reply:
> a Fontique [Collection]?

Well yes. Although there's also a fontique SourceCache. I'm not quite sure what the purpose of the SourceCache is.